apps/sms/services/js/messaging/messaging_client.js

```diff
@@ -0,0 +1,54 @@
+/* global bridge */
+
+(function(exports) {
+  'use strict';
+
+  /**
+   * Name of the service that is responsible for messaging functionality.
+   * @type {string}
+   */
+  const SERVICE_NAME = 'messaging-service';
+  const TIMEOUT = 5000;
```

Comment:
ah no, I understand now, it's the "real" network.

so 5sec is not enough, I just got the issue with a small MMS. It think we should not have a timeout here.

Another possibility is to just ignore the return for the calls that might take a long time and rely on events only. But this would be for a future patch if we want to do this.

Let's just disable the timeout for now.